packages/fxa-settings/package.json

```diff
@@ -16,16 +16,42 @@
     "stop": "pm2 stop pm2.config.js",
     "delete": "pm2 delete pm2.config.js",
     "storybook": "npm run build-postcss && start-storybook -p 6008 -s .storybook/design-guide --no-version-updates",
-    "test": "SKIP_PREFLIGHT_CHECK=true rescripts test"
+    "test": "SKIP_PREFLIGHT_CHECK=true rescripts test",
+    "test:coverage": "yarn test --coverage --watchAll=false"
   },
   "jest": {
     "resetMocks": false,
     "collectCoverageFrom": [
       "src/**/*",
       "!src/**/*.stories.*",
       "!src/**/mocks.*",
-      "!src/**/*.svg"
-    ]
+      "!src/**/*.svg",
```

Comment:
Was it including SVGs? Could whitelist instead of blacklisting, `"src/**/*.{js,jsx,ts,tsx}",` on the first line

Reply:
That was already there... I can clean this up though.

Reply:
@jodyheavener This was actually already in place, and isn't really part of the PR (the line change is just a trailing comma). I can clean this up though... To be honest, it seems unneeded, since checks should only be done on code, but I can double check this. 

I suspect whitelisting won't do what you want in conjunction with global coverage, but I guess this is what we are hashing out, at the moment..